v5/owner.apib

```diff
@@ -0,0 +1,449 @@
+     + Attributes (Image)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
+### Create a new Image [POST /owner/parking-spaces/{parking_space_id}/images]
++ Request (application/json)
+    + Attributes
+        + `content`: `w6JQTkcNChoNCgAAAA0KSUhEUgAAAMK3AAAAwrcIAwA...` (string, required) - base64 encoded image content
+
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 201 (application/json)
+    + Attributes (Image)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Attributes (Api Error)
+
+### Delete an Image [DELETE /owner/parking-spaces/{parking_space_id}/images/{image_id}]
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + image_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the image id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200
+    + Attributes (Image)
+
++ Response 404
+Returned if parking space or image was not found or if the parking space doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if an error occurred.
+    + Attributes (Api Error)
+
+### List periods [GET /owner/parking-spaces/{parking_space_id}/periods]
+Returns all periods for one parking space
+
++ Parameters
+    + parking_space_id: `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 200 (application/json)
+     + Attributes (Period)
+
++ Response 404
+Returned if parking space was not found or it doesn't belong to the user.
+    + Body
+
+### Create a new Period [POST /owner/parking-spaces/{parking_space_id}/periods]
++ Request (application/json)
+    + Attributes
+        + `date_start`: `2017-12-15` (string) - can be null
+        + `date_end`: `null` (string) - can be null
+
+    + Headers
+
+            Authorization: Basic MDEyMzQ1Njc4YWJjZGVmOnBhcmt1
+
++ Response 201 (application/json)
+    + Attributes (Period)
+
++ Response 404
+Returned if the parking space was not found or it doesn't belong to the user.
+    + Body
+
++ Response 422
+Returned if the period has overlapping dates or an error occurred.
+    + Attributes (Api Error)
+
+### Update a Period [PUT /owner/parking-spaces/{parking_space_id}/periods/{period_id}]
+Updates the period by setting the values of the parameters passed.
+
++ Parameters
+    + parking_space_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+    + period_id : `00cd7cfd-e42d-11e2-8bf1-8a83f3373875` (string, required) - the parking space id
+
++ Request  (application/json)
+    + Attributes
+        + `date_start`: `2017-12-15` (string) - can be null
```

Comment:
also update this with type and description, just as above

Reply:
Done